core/src/main/resources/core-reference.conf

```diff
@@ -43,3 +43,60 @@ crossdata-core.catalog.caseSensitive = true
 #crossdata-core.catalog.zookeeper.retryAttempts = 5
 #crossdata-core.catalog.zookeeper.retryInterval = 10000
 
+
+
+#####################################
+#                                   #
+#      Streaming configuration      #
+#                                   #
+#####################################
+
+####### StreamingCatalog ###########
+crossdata-core.streaming.catalog.class="org.apache.spark.sql.crossdata.catalog.ZookeeperStreamingCatalog"
+crossdata-core.streaming.catalog.zookeeper.connectionString = "localhost:2181"
+crossdata-core.streaming.catalog.zookeeper.connectionTimeout = 15000
+crossdata-core.streaming.catalog.zookeeper.sessionTimeout = 60000
+crossdata-core.streaming.catalog.zookeeper.retryAttempts = 5
+crossdata-core.streaming.catalog.zookeeper.retryInterval = 10000
+
+
+
+####### KafkaReceiverOptions ###########
+crossdata-core.streaming.receiver.kafka.connection = "localhost:2181:9092"
+#crossdata-core.streaming.receiver.kafka.numPartitions = "1"
+crossdata-core.streaming.receiver.storageLevel = "MEMORY_AND_DISK_SER"
+#crossdata-core.streaming.receiver.kafka.options.someKey = "someValue"
+
+####### StreamingOptions ###########
+crossdata-core.streaming.atomicWindow = "5"
+crossdata-core.streaming.maxWindow= "10"
+crossdata-core.streaming.outputFormat = "JSON"
+#The checkpoint directory will be checkpointDirectory/ephemeralTableName
+crossdata-core.streaming.checkpointDirectory = "/tmp/spark"
+crossdata-core.streaming.sparkHome = "/home/darrollo/Escritorio/spark-1.5.2-bin-hadoop2.4"
+crossdata-core.streaming.appJar = "/home/darrollo/Projects/crossdata/streaming/target/crossdata-streaming-1.2.0-SNAPSHOT-jar-with-dependencies.jar"
+#crossdata-core.streaming.jars = []
+#crossdata-core.streaming.appJar = "/home/darrollo/Projects/crossdata/streaming/target/crossdata-streaming-${project.version}-jar-with-dependencies.jar"
```

Comment:
Could we provide a more generic path? Something like "$HOME/crossdata/streaming/target/crossdata-streaming-${project.version}-jar-with-dependencies.jar" or similar